lib/routing-mixin.js

```diff
@@ -1,10 +1,10 @@
-import RouteTreeNode from './route-tree-node.js';
-import RouteData from './route-data.js';
-import {Context} from './page.js';
-import BasicRoutingInterface from './routing-interface.js';
+import RouteTreeNode from "./route-tree-node.js";
+import RouteData from "./route-data.js";
+import { Context } from "./page.js";
+import BasicRoutingInterface from "./routing-interface.js";
 
 /**
- * @param {function(new:HTMLElement)} Superclass
+ * @param {CustomElementConstructor} Superclass
```

Comment:
I don't think this is going to work for closure compiler, but I'm guessing typescript has some complaint about the type you're replacing here?